src/platform/utilities/cerner/index.js

```diff
@@ -6,10 +6,14 @@ export const isCernerLive = !environment.isProduction();
 
 export const getCernerURL = path => {
   const root = environment.isProduction()
-    ? 'https://patientportal.myhealth.va.gov'
-    : 'https://ehrm-va-test.patientportal.us.healtheintent.com';
+    ? `https://patientportal.myhealth.va.gov/clear-session?to=${encodeURIComponent(
+        'https://patientportal.myhealth.va.gov',
+      )}`
+    : `https://ehrm-va-test.patientportal.us.healtheintent.com/clear-session?to=${encodeURIComponent(
+        'https://ehrm-va-test.patientportal.us.healtheintent.com',
+      )}`;
 
-  return `${root}${path}`;
+  return `${root}${encodeURIComponent(path)}`;
```

Comment:
@kelsonic won't this put the query parameter before the path?  For example...
```
https://patientportal.myhealth.va.gov/clear-session?to=https%3A%2F%2Fpatientportal.myhealth.va.gov/sub/path
```

I think you may need to include the path on lines 10 and 13 and omit from 16